src/test/java/com/google/api/generator/gapic/composer/grpc/goldens/GrpcTestingStub.golden

```diff
@@ -220,9 +227,18 @@ public class GrpcTestingStub extends TestingStub {
             .setMethodDescriptor(getTestMethodDescriptor)
             .setParamsExtractor(
                 request -> {
-                  ImmutableMap.Builder<String, String> params = ImmutableMap.builder();
-                  params.put("name", String.valueOf(request.getName()));
-                  return params.build();
+                  RoutingHeaderParamsBuilder builder = new RoutingHeaderParamsBuilder();
+                  if (request.getName() != null) {
+                    builder.add(
+                        String.valueOf(request.getName()), "rename", PATH_TEMPLATE_GETTEST_0);
```

Comment:
I thought the same thing too, but we will have to change the method signature to `Object` and cast it to String in the builder, which is not something I prefer either. That being said, I'm considering adding the type check for String so that we don't have to do this kind of casting, I'll start a new thread and discuss with you guys.

Reply:
Removed `String.valueOf()` since the validation of String type for field is added.